forgerock-auth/src/test/java/org/forgerock/android/auth/FRUserMockTest.java

```diff
@@ -973,7 +961,7 @@ public void testRequestInterceptor() throws InterruptedException, ExecutionExcep
         Assertions.assertThat(result.get("EXCHANGE_TOKEN").second).isEqualTo(1);
         Assertions.assertThat(result.get("REVOKE_TOKEN").second).isEqualTo(1);
         Assertions.assertThat(result.get("LOGOUT").second).isEqualTo(1);
-        Assertions.assertThat(result.get("END_SESSION").second).isEqualTo(1);
+        //Assertions.assertThat(result.get("END_SESSION").second).isEqualTo(1);
```

Comment:
Is this comment intentional or left here accidentally ?

Reply:
Good Catch, that is intentional, but the END_SESSION interceptor test is missing. So I have added a test case to test END_SESSION action.